apps/system/js/app_install_manager.js

```diff
@@ -243,22 +242,28 @@
       var id = detail.id;
 
       if (manifest.size) {
+        this.size.removeAttribute('data-l10n-id');
         this.size.textContent = this.humanizeSize(manifest.size);
       } else {
-        this.size.textContent = _('size-unknown');
+        this.size.setAttribute('data-l10n-id', 'size-unknown');
```

Comment:
will it be translated on the fly by mozl10n when new data-l10n-id attribute is appended? or we could use navigator.mozL10n.setAttributes to set string for safety